.changeset/kind-crews-hear.md

```diff
@@ -0,0 +1,7 @@
+---
+'@clerk/backend': patch
+---
+
+Allow to create and update SAML connections with organization IDs.
+
+When users sign in or up using an organization's SAML connection, they're automatically added as members of that organization. For more information, refer to our docs: [TODO - Add docs link]
```

Comment:
I'll only merge this once the docs get deployed in order to reference it here

Reply:
I just updated this with what the docs link will be. We'll still avoid releasing this until the docs are live!